### ISSUE TITLE
Bugs in Local preferences in cooperation with screen readers

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#1123]
**Reported by krzyszp on 23 May 41858558 18:17 UTC**
In Local preferences (advanced view) labels for text fields are not readable by NVDA (screen reader). (Manager 6.10.13)

Migrated-From: http://boinc.berkeley.edu/trac/ticket/1150

**Comment by ChristianBeer** [comment]
related to #1088

**Comment by Ageless93** [comment]
Can someone check this with BOINC version 7.6.33 or newer, please?
